addons/mrp_subcontracting/models/stock_move.py

```diff
@@ -216,9 +216,13 @@ def _action_confirm(self, merge=True, merge_into=False):
             bom = move._get_subcontract_bom()
             if not bom:
                 continue
+            company_id = move.company_id
```

Comment:
```suggestion
            company = move.company_id
```
`_id` would mean that the variable contains the id, here you store the record

Reply:
> `_id` would mean that the variable contains the id, here you store the record Same for all next variables you have declared

The code has been updated. :)